packages/orchestrator-ui-components/src/components/WfoTree/WfoTreeNode.tsx

```diff
@@ -44,10 +44,7 @@ export const WfoTreeNode: FC<WfoTreeNodeProps> = ({
     const expanded = expandedIds.includes(item.id);
     const selected = selectedIds.includes(item.id);
 
-    let expandIcon = expanded ? 'arrowDown' : 'arrowRight';
-    if (item.id === 0) {
```

Comment:
What did this condition do and are we sure you don't need it anymore? 
It seems it toggles the exapandIcon prop only for the first item which seems significant